XTRABYTES/XCITE
# Add proof-of-concept boilerplate for zendesk integration

- Add a help/support button top right
- Opens an embedded browser targetting support.xtrabytes.global
- Add button at the top of the browser to trigger Zendesk OAuth workflow and dump any tickets to the QT console.

Testing:
- Hit top right (?) button
- Hit 'ACCESS ZENDESK' button
- Choose to Sign up to xtrabytes (create a new account for testing)
- Finish email verification in normal browser
- Login to Zendesk in XCITE
- Hit 'Submit a request' at the top of the support page in XCITE
- Use "Test XCITE integration" as the subject so the support guys know to ignore it, submit
- Hit 'ACCESS ZENDESK'
- Should dump subject/description of your ticket to the console
